docker/environments/build.sh

```diff
@@ -29,3 +29,6 @@ docker buildx build --platform linux/amd64,linux/arm64,linux/386,linux/ppc64le -
 
 echo 'Ruby 3.0...'
 docker buildx build --platform linux/amd64,linux/arm64,linux/386,linux/ppc64le -t appwrite/env-ruby-3.0:1.0.0 ./docker/environments/ruby-3.0/ --push
+
+echo 'Dart 2.10.4...'
```

Comment:
It will be better to use minor version (2.10) and not full version (2.10.4), to avoid too many supported envs.